### PR TITLE
Remove command prompts and split command outputs

### PR DESCRIPTION
**Pull request by makocchi-git**
- follow the style guide

### CODE REVIEW

**Comment by makocchi-git** [comment]
[preview is here](https://deploy-preview-13015--kubernetes-io-master-staging.netlify.com/docs/tasks/access-application-cluster/access-cluster/)
[original](https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/)

File: content/en/docs/tasks/access-application-cluster/access-cluster.md [hunk 1]
```diff
@@ -65,7 +65,10 @@ Then you can explore the API with curl, wget, or a browser, replacing localhost
 with [::1] for IPv6, like so:
 
 ```shell
-$ curl http://localhost:8080/api/
+curl http://localhost:8080/api/
+```
+
```
Comment (DanyC97) [inline content/en/docs/tasks/access-application-cluster/access-cluster.md]: please add The output is similar to this: as per the guidelines before any output

File: content/en/docs/tasks/access-application-cluster/access-cluster.md [hunk 2]
```diff
@@ -76,16 +79,17 @@ $ curl http://localhost:8080/api/
...
-$ SECRET_NAME=$(kubectl get secrets | grep ^default | cut -f1 -d ' ')
-$ TOKEN=$(kubectl describe secret $SECRET_NAME | grep -E '^token' | cut -f2 -d':' | tr -d " ")
+APISERVER=$(kubectl config view --minify | grep server | cut -f 2- -d ":" | tr -d " ")
+SECRET_NAME=$(kubectl get secrets | grep ^default | cut -f1 -d ' ')
+TOKEN=$(kubectl describe secret $SECRET_NAME | grep -E '^token' | cut -f2 -d':' | tr -d " ")
 
-$ curl $APISERVER/api --header "Authorization: Bearer $TOKEN" --insecure
+curl $APISERVER/api --header "Authorization: Bearer $TOKEN" --insecure
+```
+
```
Comment (DanyC97) [inline content/en/docs/tasks/access-application-cluster/access-cluster.md]: please add The output is similar to this: as per the guidelines before any output

File: content/en/docs/tasks/access-application-cluster/access-cluster.md [hunk 3]
```diff
@@ -103,11 +107,14 @@ $ curl $APISERVER/api --header "Authorization: Bearer $TOKEN" --insecure
...
 ```shell
-$ APISERVER=$(kubectl config view --minify -o jsonpath='{.clusters[0].cluster.server}')
-$ SECRET_NAME=$(kubectl get serviceaccount default -o jsonpath='{.secrets[0].name}')
-$ TOKEN=$(kubectl get secret $SECRET_NAME -o jsonpath='{.data.token}' | base64 --decode)
+APISERVER=$(kubectl config view --minify -o jsonpath='{.clusters[0].cluster.server}')
+SECRET_NAME=$(kubectl get serviceaccount default -o jsonpath='{.secrets[0].name}')
+TOKEN=$(kubectl get secret $SECRET_NAME -o jsonpath='{.data.token}' | base64 --decode)
+
+curl $APISERVER/api --header "Authorization: Bearer $TOKEN" --insecure
+```
```
Comment (DanyC97) [inline content/en/docs/tasks/access-application-cluster/access-cluster.md]: please add The output is similar to this: as per the guidelines before any output

**Comment by makocchi-git** [comment]
@DanyC97 Thank you for your comments! I fixed.

**Comment by DanyC97** [comment]
thank you @makocchi-git for accepting my suggestions

/lgtm

**Comment by cody-clark** [comment]
Thanks, @makocchi-git! 

/approve

**Pull request merged**